app/components/TransactionModal.tsx

```diff
@@ -54,6 +56,18 @@ export const TransactionModal = ({
                   </p>
                 </div>
               </>
+            ) : isError ? (
+              <>
+                <IconBadge name="exclamation" colorScheme="error" />
+                <div className="flex flex-col items-center space-y-2">
+                  <p className="text-2xl font-semibold text-text-high-em">
+                    There was an error.
+                  </p>
+                  <p className="font-semibold text-center text-text-low-em text-md max-w-80">
+                    Unfortunately the transaction was not completed.
+                  </p>
+                </div>
+              </>
             ) : (
```

Comment:
WDYT about simplifying the readability a bit by using short-circuit instead of multi-ternary operators? Something like:
```ts
{isError && ()}
{!isError && ()}
```

Reply:
We can do that 👍